src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 50px auto;
+  gap: 1px;
+  width: ($cell-size * $cell-rows) + ($cell-rows * $cell-gap);
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 0.5s;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    border: 1px solid #000;
+    box-sizing: border-box;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      cursor: pointer;
+    }
+
+    @for $cell-number from 1 through 31 {
+      &--#{$cell-number}::before {
```

Comment:
You should use `nth-child(#{$cell-number})::before` in this case. Because you have pseudo-class for that.